src/powerbox/dft.py

```diff
@@ -280,47 +294,3 @@ def _retfunc(ft, freq, axes, ret_cubegrid):
         grid = np.add.outer(grid, freq[i] ** 2)
 
     return ft, freq, np.sqrt(grid)
-
-
-def fftshift(x, *args, **kwargs):
-    """
-    The same as numpy, except that it preserves units (if Astropy quantities are used).
-
-    All extra arguments are passed directly to numpy's ``fftshift``.
-    """
-    out = _fftshift(x, *args, **kwargs)
-
-    return out * x.unit if hasattr(x, "unit") else out
-
-
-def ifftshift(x, *args, **kwargs):
-    """
-    The same as numpy except it preserves units (if Astropy quantities are used).
-
-    All extra arguments are passed directly to numpy's ``ifftshift``.
-    """
-    out = _ifftshift(x, *args, **kwargs)
-
-    return out * x.unit if hasattr(x, "unit") else out
-
-
-def fftfreq(N, d=1.0, b=2 * np.pi):
-    """
-    Return fourier frequencies for a box with N cells, using general Fourier convention.
-
-    Parameters
-    ----------
-    N : int
```

Comment:
Unfortunately, deleting this will change the API, so we should _try_ to leave them there. You could make them thin wrappers, e.g.:

```python
def fftshift(x, *args, **kwargs):
    backend = kwargs.get("backend", NumpyFFT())
    return backend.fftshift(x, *args, **kwargs)
```